--- librepo-1.12.1/doc/python/CMakeLists.txt.orig	2022-03-29 06:12:18.723722231 +0200
+++ librepo-1.12.1/doc/python/CMakeLists.txt	2022-03-29 17:04:23.903595141 +0200
@@ -1,5 +1,5 @@
 ADD_CUSTOM_TARGET (doc-python
-    PYTHONPATH=${CMAKE_BINARY_DIR}/librepo/python/python${PYTHON_VERSION_MAJOR} sphinx-build -E -b html
+    PYTHONPATH=${CMAKE_BINARY_DIR}/librepo/python/python${PYTHON_VERSION_MAJOR} ${SPHINX_EXECUTABLE} -E -b html
                   ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}
                   COMMENT "Building Python API documentation with Sphinx")
 
